ext/telemetry/errors.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = function(tracker) {
+  return {
+    trackUncaught: () => {
+      window.onerror = (msg, url, line, col, err) => {
```

Comment:
I'm  a little hesitant to use `process.on('uncaughtException')` since the docs recommend using it for cleanup purposes:

> The correct use of 'uncaughtException' is to perform synchronous cleanup of allocated resources (e.g. file descriptors, handles, etc) before shutting down the process. It is not safe to resume normal operation after 'uncaughtException'.

And also to prevent overwriting any changes atom has made to it internally. The [docs for `window.onerror`](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror#element.onerror_(and_window.addEventListener('error'))) make it sound like a bit more appropriate tool for error handling